output/eu_crude/analysis.py

```diff
@@ -0,0 +1,153 @@
+import sqlite3 as lite
+import sys
+import matplotlib as plt
+from pyne import nucname
+
```

Comment:
yeah... I was going to talk to  you about that.
How serious is this line too long violation?
Maybe it's just me but separating some of the lines
makes it more confusing. If this is like a non-negotiable thing I'll change!